### DISCUSSION TITLE
Document Collector Contrib Exporters/Receivers/Processors On Website

### DISCUSSION THREAD

**Discussion by austinlparker** [open-telemetry/opentelemetry.io#4301] [Content Refactoring]
Currently, there are dozens of components in https://github.com/open-telemetry/opentelemetry-collector-contrib that have somewhat exhaustive user documentation and metadata about their outputs in README.md and metadata.yml files in each subdirectory. In order to improve discoverability of these components, and to increase the usability of those components, this documentation should live on the opentelemetry.io website.

This issue was discussed at the 1/29/24 SIG meeting, and the following broad points were agreed upon:
- It is fine for this to be an issue that is progressively enhanced over time, but we should err on delivering a v0 or v1 sooner rather than later.
- Ideally, we can generate the user-facing docs from the metadata files exclusively in the future
- The actual docs should be available as part of registry cards
- A new 'integrations' landing page for the collector docs can point to the registry
- We should focus on getting 'stable' components in first as there can be a wide variance between the quality of their metadata files/readmes, but will need to do some human curation to determine which are the most critical

**Comment by theletterf** [comment]
This is a good proposal, especially the bit about automating docs generation from the metadata.yaml files.

OTOH, what's our overall stance regarding contrib? Is OpenTelemetry.io only concerned with core? Should it be?
